test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj

```diff
@@ -41,6 +44,11 @@
 (deftest ^:integration old-master-route-config
   (testing "The old map-style route configuration map still works."
     (bootstrap/with-puppetserver-running-with-mock-jrubies
+     "MOCK.TODO: discuss - this is *probably* ok because all we really care about
+     is that the webserver is accepting connections and handing them to the correct
+     ring handler.  Would be better if we could implement this to hit a clojure
+     endpoint instead of a Ruby one but I don't remember if there are any pure-clojure
+     endpoints under /puppet."
```

Comment:
I think I'm fine with that rationale as it regards this particular test, but less comfortable with the idea that that justifies having the default mock behavior return a 200, because not all tests that might be exercising that mocked code path are as inconsequential as this one.